### PR TITLE
Fix/pvc error

### PR DESCRIPTION
**Pull request by scirag**
**Description of the change**
A brief description of the change

**Benefits**
The benefits of the change

**Possible drawbacks**
Any limitations

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[chart-name] Update depedency`)

### CODE REVIEW

**Comment by scirag** [comment]
please check

**Comment by scirag** [comment]
update include pvc-name

File: charts/solr/values.yaml [hunk 1]
```diff
@@ -134,7 +134,7 @@ podAnnotations: {}
 ##
 volumeClaimTemplates:
   ## solr data Persistent Volume Storage Class
-  storageClassName: ""
```
Comment (lwj5) [inline charts/solr/values.yaml:137]: Do not hardcode

File: charts/solr/templates/storageclass.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: efs-sc
```
Comment (lwj5) [inline charts/solr/templates/storageclass.yaml:4]: Do not add a StorageClass, this should be done by the CSI driver or by the cluster-admin on the cluster directly

File: charts/solr/templates/statefulset.yaml [hunk 3]
```diff
@@ -225,7 +225,7 @@ spec:
         {{- end }}
   volumeClaimTemplates:
     - metadata:
-        name: {{ include "solr.pvc-name" . }}
```
Comment (lwj5) [inline charts/solr/templates/statefulset.yaml:228]: Do not hardcode

File: charts/solr/templates/pv.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolume
```
Comment (lwj5) [inline charts/solr/templates/pv.yaml:2]: PV should be created by the CSI driver not manually

File: charts/solr/templates/claim.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+kind: PersistentVolumeClaim
+metadata:
+  name: efs-claim
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: efs-sc
+  resources:
+    requests:
+      storage: 20Gi
```
Comment (lwj5) [inline charts/solr/templates/claim.yaml:11]: There's no need for PVC in a stateful set

File: charts/solr/templates/_helpers.tpl [hunk 6]
```diff
@@ -70,7 +70,7 @@ Create chart name and version as used by the chart label.
   Define the name of the solr PVC
 */}}
 {{- define "solr.pvc-name" -}}
-{{ printf "%s-%s" (include "solr.fullname" .) "pvc" | trunc 63 | trimSuffix "-"  }}
+{{ printf "efs-claim"  }}
```
Comment (lwj5) [inline charts/solr/templates/_helpers.tpl:73]: Do not hardcode

**Review by lwj5 — COMMENTED** [review]
Nothing to merge.

**Pull request closed without merging**